app/services/funding/update_trainee_summary_row_route_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Funding
+  class UpdateTraineeSummaryRowRouteService
+    include ServicePattern
+
+    ROUTE_TYPES = LeadPartnerTraineeSummariesImporter::ROUTE_TYPES.merge(
```

Comment:
Show this be only for a particular year

Reply:
Yes as I'm not sure if pre `2024/25` should have the new route names. 

So the idea is that if we run this now it will fill the `route_type` only the current academic year and therefore use the ITT translations. Next year if the funding team adds new routes to the CSV files we can update the code on our end and rerun the task to fill the `route_type` column for `2025/26` if we decide to use different route names than they one's that they have used in their uploads.